### DISCUSSION TITLE
Exclusion action in SOC

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#13075] [Ideas] [answered]
Hi!

Great that I can edit exclusions now directly in SOC 2.4,  but have you ever thought about a single action in SOC to create an exclusion based on an existing alert (in the left click context menu of an alert)? That would be so AWESOME! Something like a small pop up where a user can decide if her/she wants to disable the alert rule completly or just suppress is it for specific source and/or destination IP's would be a good start.

Cheers, Ben

**Comment by reyesj2** [comment]
Stay tuned for 2.4.70 https://www.youtube.com/watch?v=oxR4q53N6OI !

**Comment by dougburks** [comment, marked as answer]
Security Onion 2.4.70 now available including our new Detections interface and much more!
https://blog.securityonion.net/2024/05/security-onion-2470-now-available.html

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/6537e8f9-967b-47d7-bdc1-6172fb0e398a)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/98840126-cccc-452e-b186-9340a0799ce8)
